Packs/FeedDShield/Integrations/FeedDShield/FeedDShield.py

```diff
@@ -0,0 +1,49 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+
+    params['indicator_type'] = FeedIndicatorType.CIDR
+
+    params['url'] = 'https://www.dshield.org/block.txt'
+    params['ignore_regex'] = "[#S].*"
+    params['indicator'] = json.dumps({
+        "regex": r"^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\t[\d.]*\t(\d{1,2})",
+        "transform": "\\1/\\2"
+    })
+
+    fields = json.dumps({
```

Comment:
Why `json.dumps`?

Reply:
only way I was able to make it work - str changes it to single quotation mark which creates an error.